mill

```diff
@@ -2,7 +2,7 @@
 
 # Adapted from
 
-coursier_version="2.1.0-M6-53-gb4f448130"
+coursier_version="2.1.0-RC5"
```

Comment:
Not sure about this script? Is that an offical mill / millw script actually? Or just some hacked together custom script for scala-ci?

Reply:
It's a kind of customer script specific for `scala-cli`. You should also update courier version in this script. It's fine.